assets/airports/ktus.json

```diff
@@ -0,0 +1,2422 @@
+                ["aal", 11.6],
+                ["ash", 9.9],
+                ["asq", 1.7],
+                ["cpz", 1.7],
+                ["dal", 2.5],
+                ["n/fastGA", 26.4],
+                ["skw", 25.6],
+                ["swa", 14.9]
+            ]
+        },
+        {
+            "origin": "KTUS",
+            "destination": "",
+            "category": "departure",
+            "route": "KTUS.BURRO4.HOTTT",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 1.3,
+            "airlines": [
+                ["aal", 11.6],
+                ["asa", 1.7],
+                ["ash", 9.9],
+                ["asq", 1.7],
+                ["cpz", 1.7],
+                ["dal", 2.5],
+                ["n/fastGA", 26.4],
+                ["qxe", 1.7],
+                ["skw", 25.6],
+                ["swa", 14.9]
+            ]
+        },
+        {
+            "origin": "KTUS",
+            "destination": "",
+            "category": "departure",
+            "route": "KTUS.BURRO4.NOCHI",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 2.8,
+            "airlines": [
+                ["aal", 11.6],
+                ["ash", 9.9],
+                ["asq", 1.7],
+                ["cpz", 1.7],
+                ["dal", 2.5],
+                ["fdx", 2.5],
+                ["n/fastGA", 26.4],
+                ["skw", 25.6],
+                ["swa", 14.9]
+            ]
+        },
+        {
+            "origin": "KTUS",
+            "destination": "",
+            "category": "departure",
+            "route": "KTUS.TUS8.REDDY",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.5,
+            "airlines": [
+                ["aal", 11.6],
+                ["ash", 9.9],
+                ["asq", 1.7],
+                ["cpz", 1.7],
+                ["dal", 2.5],
+                ["fdx", 2.5],
+                ["n/fastGA", 26.4],
+                ["skw", 25.6],
+                ["swa", 14.9]
+            ]
+        },
+        {
+            "origin": "KTUS",
+            "destination": "",
+            "category": "departure",
+            "route": "KTUS.TUS8.PXR",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 0.3,
+            "airlines": [
+                ["aal", 11.6],
+                ["asa", 1.7],
+                ["ash", 9.9],
+                ["asq", 1.7],
+                ["cpz", 1.7],
+                ["dal", 2.5],
+                ["n/fastGA", 26.4],
+                ["qxe", 1.7],
+                ["skw", 25.6],
+                ["swa", 14.9]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KTUS",
+            "category": "arrival",
+            "route": "GBN.DINGO5.KTUS",
+            "altitude": 15000,
+            "speed": 320,
```

Comment:
Horizon (QXE) operates a whole bunch of DH8s, the earlier series of which would not be able to keep this speed.